frontend/src/App.js

```diff
@@ -3,24 +3,22 @@ import { Switch, Route } from 'react-router-dom';
 import Homepage from './containers/Homepage/Homepage.js';
 import LoginPage from './containers/LoginPage/LoginPage.js';
 import SignUp from './components/SignUp/SignUp.js';
-import UsersAll from './components/AdminControls/UsersAll.js';
-import User from './components/AdminControls/User.js';
 import history from './history.js';
- 
+import AdminPanel from './containers/AdminPanel/AdminPanel'
+
 class App extends Component {
   render() {
     return (
       <div>
         <Switch>
-          <Route exact path='/' component={Homepage}></Route>
-          <Route path='/login' component={LoginPage}></Route>
-          <Route path='/signup' component={SignUp}></Route>
-          <Route exact path='/users' history={history} render={(props) => <UsersAll {...props} />}></Route>
-          <Route exact path='/users/:id' history={history} render={(props) => <User {...props} />}></Route>
+          <Route exact path="/" component={Homepage} />
+          <Route path="/login" component={LoginPage} />
+          <Route path="/signup" component={SignUp} />
+          <Route path="/admin" history={history} component={AdminPanel} />} />
```

Comment:
We're removing history in another branch, and you're not using it in your component, so probably don't need to pass it here.